### PR TITLE
[DC-2150] Convert only CDM tables to v531

### PR DESCRIPTION
**Pull request by nishanthpp93**

### CODE REVIEW

File: data_steward/tools/migrate_cdm52_to_cdm531.py [hunk 1]
```diff
@@ -76,13 +80,16 @@ def get_upgrade_table_query(client, dataset_id, table_id, hpo_id=None):
     try:
```
Comment (mark-velez) [inline data_steward/tools/migrate_cdm52_to_cdm531.py:80]: nit: consider removing the try catch so it can fail loudly in cases we didn't anticipate

**Review by mark-velez — APPROVED** [review]
Thanks @nishanthpp93 . This migration script is only temporary; I think this gets the job done without overthinking it. Added a nitpick which you can ignore if you wish.

File: data_steward/tools/migrate_cdm52_to_cdm531.py (hunk 1 above)
Comment (nishanthpp93) [reply data_steward/tools/migrate_cdm52_to_cdm531.py:80]: I think this makes sense since we are filtering down the list to only CDM

Comment (nishanthpp93) [reply data_steward/tools/migrate_cdm52_to_cdm531.py]: It looks like this is useful for tables we don't have schemas for, like data_browser tables etc., so keeping this.

**Pull request merged**